nemo_aligner/models/nlp/gpt/megatron_gpt_ppo_actor.py

```diff
@@ -98,6 +99,7 @@ def __init__(self, cfg: DictConfig, trainer: Trainer):
                 use_greedy=self.cfg.ppo.sampling_params.get("use_greedy", False),
                 tokenizer=self.tokenizer,
                 seed=self.cfg.ppo.trt_llm.get("seed", self.cfg.seed),
+                trt_model_dir=self.cfg.ppo.get("trt_model_dir", "/tmp/trt_llm_model"),
```

Comment:
it will be nice if you also added this onto the PPO actor yaml file too, just so users know it's possible to configure

Reply:
Oops, I think that was dropped. I'll add it back

Reply:
Added back in: 73f737180aeb5d4afc72140fc66f33d1544693b0

Reply:
So I looked everywhere and it doesn't appear to be used